src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlDataReader.cs

```diff
@@ -2045,6 +2046,7 @@ override public TextReader GetTextReader(int i)
         }
 
         /// <include file='../../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlDataReader.xml' path='docs/members[@name="SqlDataReader"]/GetChar/*' />
+        [EditorBrowsableAttribute(EditorBrowsableState.Never)]
```

Comment:
I'm not sure that this should be moved across to netcore.

Reply:
For alignment purpuses, is it OK to remove it from netfx then?

Reply:
I've just checked the ref projects; this is present in both [netfx](https://github.com/dotnet/SqlClient/blob/main/src/Microsoft.Data.SqlClient/netfx/ref/Microsoft.Data.SqlClient.cs#L1294) and [netcore](https://github.com/dotnet/SqlClient/blob/main/src/Microsoft.Data.SqlClient/netcore/ref/Microsoft.Data.SqlClient.cs#L1282). I personally think it's more likely that someone missed a spot when adding the attribute before the source code was uploaded. In light of the ref projects, perhaps we should add the EditorBrowsableAttribute to netcore?

Reply:
> I think this is the sort of thing that for merging purposes we should leave alone. In the merged version, wrap it in a `#if NETFRAMEWORK`. While it's not the cleanest solution, we have precedent for doing this in other merges, and it is the safest change.

What about a combination, wrapping it in a #IF NETFRAMEWORK in both versions?

Reply:
I think if it's present in both ref projects, we should probably have it in both netfx and netcore implementation projects. I officially rescind my comment :)